styles/controllers/grid/queries.less

```diff
@@ -27,12 +27,9 @@
         margin-bottom: @double;
     }
 
-    .add_note {
-        text-align: right;
+    .queryEditButtons {
 
-        a {
-            &:extend(.pkp_button all);
-        }
+        display: flex;
```

Comment:
So, we haven't used any `flex` yet in the backend. We're probably safe from a browser perspective now (looks like IE10 is the most recent browser to require a prefix), so you can leave it if you want. But another option that should work is this:

```css
.queryEditButtons {
  > button {
    display: inline-block;
  }
}
```

Reply:
I see that you will still need a way to initialize it correctly in the template below. You could use a `style="display: none;"` attribute. It's not pretty but it's ok enough since it is slated for replacement eventually.

Reply:
Oh, and it looks like they may be `a` not `button`...

Reply:
ah, the flex was probably left there because I originally tried to position the buttons to opposite corners and flex offered an easy way of doing that. I changed that to inline-block.

It would be good to have a fixed way of placing buttons like that in the forms, I mean a ready set of css classes. Maybe there is already?